grpc-interface-support/src/python.py

```diff
@@ -41,7 +45,49 @@ def get_required_sdk_version_python() -> str:
     return sdk_version
 
 
+def get_template_dir() -> str:
+    return os.path.join(
+        get_package_path(), "grpc-interface-support", "data", "templates", "python"
+    )
+
+
+class GrpcCodeExtractor:
+    """
+    Provides methods for extracting code from generated gRPC python files.
+    """
+
+    def __init__(
+        self, proto_file: ProtoFileHandle, base_path: str, source_path: str = "."
+    ):
+        self.__proto_file = proto_file
+        self.__base_path = base_path
+        self.__source_path = source_path
+
+        self.grpc_source_path = os.path.join(
+            self.__base_path,
+            self.__source_path,
+            f"{self.__proto_file.get_service_name().lower()}_pb2_grpc.py",
+        )
+
+    def create_source_stub_code(self) -> List[str]:
+        service_name = to_camel_case(self.__proto_file.get_service_name())
+
+        source_content: List[str] = capture_area_in_file(
+            open(self.grpc_source_path, encoding="utf-8"),
+            f"class {service_name}Servicer(object):",
+            f"def add_{service_name}Servicer_to_server(servicer, server):",
```

Comment:
client to server? I do not understand Servicer tbh what it is. I think client and server is more clear.

Reply:
See comment above. I personally don't mind. The "Servicer" naming was already there :)

https://grpc.io/docs/languages/python/basics/